oasis/radare2

```diff
@@ -0,0 +1,15 @@
+Flag radare2
+  Description: Build radare2 plugin
+  Default: false
+
+Library objdump_plugin
+  Build$:           flag(everything)
```

Comment:
the expression should `flag(everything) || flag(radare2)`

Reply:
sorry for having so much mistakes, I'm not very familiar with BAP's build system and was a bit hasty writing the oasis file.

Reply:
no worries at all! I myself also usually copy-paste and edit, and keep making this kind of errors.